SayTheirNames/Source/Service/Service.swift

```diff
@@ -42,5 +42,14 @@ final class Service: Servicing {
         Log.mode = .all
         Log.print("STN Version: \(Bundle.versionBuildString)")
         Log.print("Starting Services")
+        
+        self.network.fetchPeopleByName("george") { (result) in
```

Comment:
should we remove this?

Reply:
Yeah I pushed after you started the review.